docs_source_files/content/grid4/_index.de.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Selenium Grid"
+chapter: true
+weight: 9
+---
+
+# Selenium Grid 4
+
+{{% notice info %}}
+<i class="fas fa-language"></i> Diese Seite wird von Englisch 
+auf Deutsch übersetzt. Sprichst Du Deutsch? Hilf uns die Seite 
+zu übersetzen indem Du uns einen Pull Reqeust schickst!
+ {{% /notice %}}
+
+_Selenium Grid 4_ is a smart proxy server
+that allows Selenium tests to route commands to remote web browser instances.
+Its aim is to provide an easy way to run tests in parallel on multiple machines.
+
+With Selenium Grid 4,
+one server acts as the hub that routes JSON formatted test commands
+to one or more registered Grid 4 nodes.
```

Comment:
Can we remove '4' here. While i read it makes me a bit confusion like `Grid for nodes` and later i get it as Grid 4 as this document is for Grid 4. As we are already mentioning Selenium Grid 4 in title, Can we simply say Grid nodes/ registered nodes ?